releases/1.4/kubeflow-bundle.yaml

```diff
@@ -0,0 +1,92 @@
+bundle: kubernetes
+name: kubeflow
+applications:
+  admission-webhook: { charm: admission-webhook, channel: 1.4/stable, scale: 1 }
+  argo-controller: { charm: argo-controller, channel: 3.1.8/stable, scale: 1 }
+  argo-server: { charm: argo-server, channel: 3.0.1/stable, scale: 1 }
```

Comment:
These 2 charms being different versions will probably be an issue, we should merge this open [PR](https://github.com/canonical/argo-operators/pull/23/commits) and commit to `3.1.8`.